core/lib/zksync_core/src/lib.rs

```diff
@@ -1100,15 +1100,15 @@ async fn run_http_api<G: L1GasPriceProvider>(
     }
     namespaces.push(Namespace::Snapshots);
 
-    let last_miniblock_pool = ConnectionPool::singleton(postgres_config.replica_url()?)
+    let updaters_pool = ConnectionPool::singleton(postgres_config.replica_url()?)
```

Comment:
I think you should create pool with 2 connections, so that sealed_miniblock and nonce_sweeper tasks don't block each other

Reply:
Well, if nonce_sweeper query takes much time for some reason it will drastically affect sealed_miniblock task which is bad

Reply:
I would insist on either limiting max number of accounts stored in nonce cache or having separate DB connection for each update task. First would be better IMO